src/gradient-common.ts

```diff
@@ -7,6 +7,11 @@ const directionProperty = new Property<Gradient, string>({
   defaultValue: "to bottom"
 });
 
+const radiusProperty = new Property<Gradient, string>({
+  name: "radius",
+  defaultValue: 0
```

Comment:
On line 10 the property is defined as type `string`, so TSC doesn't like this default - but no worries, I'll merge and adapt it.